shared/src/apiConsumer/ApiConsumer.ts

```diff
@@ -141,3 +141,16 @@ export const createApiConsumerParamsFromApiConsumer = (
   contact: apiConsumer.contact,
   description: apiConsumer.description,
 });
+
+export const findSubscribedEventFromId = (
+  apiConsumer: ApiConsumer,
+  subscriptionId: ApiConsumerSubscriptionId,
+): SubscriptionEvent | undefined => {
+  for (const rightName of apiConsumerRightNames) {
```

Comment:
est-ce qu'on peut le faire sans boucle for ?

Reply:
@enguerranws tu as une suggestion ?
car le `for` permet d'interrompre la boucle quand je fais un `return` plus bas